Misc/NEWS.d/next/Core and Builtins/2020-09-07-09-45-47.bpo-1635741.QuDIut.rst

```diff
@@ -0,0 +1,2 @@
+Convert the :mod:`_sha256` extension module types to heap types
+(:pep:`489`).

```

Comment:
Hum, PEP 489 is unrelated to heap types: https://www.python.org/dev/peps/pep-0489/

And the _sha256 module already uses the multi-phase init API (PEP 489). Using static types with the multi-phase init API is unsafe.

```suggestion
Convert the :mod:`_sha256` extension module types to heap types.
```